src/doc/rustc/src/lints/levels.md

```diff
@@ -33,6 +34,41 @@ But this code violates the `missing_docs` lint.
 These lints exist mostly to be manually turned on via configuration, as we'll
 talk about later in this section.
 
+## expect
+
+Sometimes, it can be helpful to suppress lints, but at the same time ensure that
+the code in question still emits them. The 'expect' level does exactly this, with
+it a lint can be expected. If the lint in question is not an emitted, a new lint
+`unfulfilled_lint_expectation` at the attribute notifying you that the expectation,
+is no longer fulfilled.
+
+```rust
+fn main() {
+    #[expect(unused_variables)]
+    let unused = "Everyone ignores me";
+
+    #[expect(unused_variables)]
+    let used = "I'm usefull";
```

Comment:
```suggestion
    let used = "I'm useful";
```

Reply:
Me and my spelling, a never-ending story. Soon I'll have finished season two, you can preorder a copy if you'd like xD